src/_data/LiveData.js

```diff
@@ -68,6 +69,7 @@ const initAuthorized = (authData, store) => {
     api.getPayoutCurrencies();
     api.subscribeToBalance();           // some call might fail due to backend overload
     api.subscribeToAllOpenContracts();
+    api.getPaymentAgentsForCountry('id');
```

Comment:
This is hardcoded just for testing, backend have not support country_code

We should merge after that